src/fiatconnect/saga.ts

```diff
@@ -67,6 +76,23 @@ export function* watchFetchFiatConnectQuotes() {
   yield takeLeading(fetchFiatConnectQuotes.type, handleFetchFiatConnectQuotes)
 }
 
+export function* handleFetchFiatConnectProviders() {
+  const account: string = yield select(currentAccountSelector)
+  try {
+    if (!account) {
+      throw new Error('Cannot fetch fiatconnect providers without an account')
+    }
+    const providers: FiatConnectProviderInfo[] = yield call(getFiatConnectProviders, account)
```

Comment:
seems like the logic is split between fiatconnect/index.ts and fiatconnect/saga.ts for this and fetch quotes. Wonder if we should consolidate in a single place. Related to https://github.com/valora-inc/wallet/pull/2687#discussion_r921410188

Reply:
I think that its not worth refactoring, but good to consider the logic being together for the rest of the implementation.